sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Pickup Based Voxel Grid Downsample Filter Node",
+  "type": "object",
+  "definitions": {
+    "pickup_based_voxel_grid_downsample_filter": {
+      "type": "object",
+      "properties": {
+        "voxel_size_x": {
+          "type": "number",
+          "description": "voxel size along x-axis [m]",
```

Comment:
```suggestion
          "description": "voxel size along the x-axis [m]",
```

Reply:
This also applies to the other axes. I think there are more occurrences of this on other of your PRs